sdk/config_helpers.go

```diff
@@ -71,6 +79,30 @@ func (dm DirectoryMap) appendFile(dir string, info fs.FileInfo) error {
 	return dm.appendFileWithProto(dir, fileProto)
 }
 
+func getLineCount(path string) (int, error) {
+	reader, err := os.Open(path)
+	if err != nil {
+		return 0, fmt.Errorf("failed to read file(%s) while trying to get lineCount: %v", path, err)
+	}
+	defer reader.Close()
+
+	buf := make([]byte, 32*1024)
+	count := 0
+	lineSep := []byte{'\n'}
+
+	for {
+		c, err := reader.Read(buf)
+		count += bytes.Count(buf[:c], lineSep)
+
+		switch {
+		case err == io.EOF:
+			return count, nil
+		case err != nil:
+			return count, fmt.Errorf("failed to read file(%v): %v", path, err)
```

Comment:
Think you could make a minor update to error message to replace %v with %s for the path
`fmt.Errorf("failed to read file(%s): %v", path, err)`